docs/src/content/docs/ref/core-advanced.mdx

```diff
@@ -251,6 +253,80 @@ nativeReserve(ton("0.1"), ReserveExact | ReserveBounceIfActionFail);
 //            amount of nanoToncoins to reserve
 ```
 
+## parseStdAddress
```

Comment:
We should mention the connection between `newAddress` and `parseStdAddress`:

```tact
let stdAddr = parseStdAddress(addr.asSlice());
newAddress(stdAddr.workchain, stdAddr.address) == addr;
```

Reply:
Added a note by the end of the usage example